driver_scripts/setup_ocean_region_groups.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
 
 """
-This script creates 6 major ocean basins and 5 regions used for computing the
-meridional overturning circulation (MOC).  No arguments are required. The
-optional --plot flag can be used to produce plots of each basin or region.
+This script creates 7 major ocean basins, 5 regions used for computing the
```

Comment:
I would add more detail here: "This script creates the following ocean region groups: i) OceanBasinRegionsGroup, which includes the Atlantic, Pacific, Indian, Arctic, Southern Ocean, Equatorial (global 15S-15N), and Mediterranean basins; ii) MOCBasinRegionGroup, which includes five regions used for computing the meridional overturning circulation (MOC) and meridional heat transport (MHT); and iii) NinoRegionGroups, which includes the Nino3, Nino4, and Nino3.4 regions."